src/main/java/uk/gov/companieshouse/pscfiling/api/controller/impl/BaseFilingControllerImpl.java

```diff
@@ -19,6 +19,9 @@
 import uk.gov.companieshouse.pscfiling.api.service.TransactionService;
 import uk.gov.companieshouse.sdk.manager.ApiSdkManager;
 
+/**
+ * Base class responsible for retrieving/updating transaction resources and handling patch failures
+ */
```

Comment:
reference to Patch here?  Is that right/needed?

Reply:
Updated to remove reference to patch.